source/hardware/controllers/gemini_g4v.rst

```diff
@@ -0,0 +1,247 @@
+Gemini G4V
+=====================
+
+.. figure:: ../../_static/controllers/gemini_g4v_topview.jpg
+   :align: center
+   :width: 100%
+   :figwidth: 100%
+   :alt: Behringer CMD Micro
+   :figclass: pretty-figures
+
+The Gemini G4V is a 2 deck controller that supports 4 virtual decks, a 4 way mixer and has a built-in 4 channel USB soundcard.
+It includes a Microphone input line, an Aux input line and an Booth output line. This lines are not going through the soundcard but mixed directly at the controller.
+They are not accessible nor controlled by Mixx so they will not appear in any recording nor is possible to control them via the Mixxx screen interface.
+
+
+-  `Manufacturer’s product page <http://geminisound.com/product/g4v>`__
+-  `Forum thread <https://www.mixxx.org/forums/viewtopic.php?f=6&t=12919>`__
+-  `Manufacturer’s User Guide <https://www.manualslib.com/manual/826563/Gemini-G4v.html>`__
+
+.. versionadded:: 2.3
+
+Mixxx sound hardware preferences
+--------------------------------
+
+========== =============
+Output     Channel
+========== =============
+Master     Channel 1 - 2
+Headphones Channel 3 - 4
+========== =============
+
+Mapping Description
+-------------------
+
+Most of the board's controls work as described in the G4V's manual, with a few exceptions. Snapshots of the board's labeling from the Manual are included below:
+
+
+.. figure:: ../../_static/controllers/gemini_g4v_controls.png
+   :align: center
+   :width: 100%
+   :figwidth: 100%
+   :alt: Behringer CMD Micro
+   :figclass: pretty-figures
+
+.. figure:: ../../_static/controllers/gemini_g4v_frontview.jpg
+   :align: center
+   :width: 100%
+   :figwidth: 100%
+   :alt: Behringer CMD Micro
+   :figclass: pretty-figures
+
```

Comment:
These should be vector graphics if possible. The labels are blurry and barely readable, and raster images are hard to edit.
Use can use these: [gemini_g4v.zip](https://github.com/mixxxdj/manual/files/6164503/gemini_g4v.zip)

I also suggest make separate sections for top, front and rear and make the sections below subsections of the section that the control belongs to.

Reply:
Also, please fix the `:alt:` label ;-)